packages/css/src/components/header/README.md

```diff
@@ -2,13 +2,22 @@
 
 # Header
 
-Arranges the City’s logo, the name of the application, and a page menu.
+Displays the City’s logo and a navigation menu at the top of every page.
```

Comment:
It doesn't necessarily show a navigation menu, right?

Reply:
mijn.amsterdam.nl and meldingen.amsterdam.nl currently do not have a navigation menu. I think the way it's currently phrased can imply that a navigation menu is mandatory.